config/hooks/pre-commit

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo -e "\n================ Running pre-commit checks ================"
+
+# Update hooks and run checkstyle on all code and AsciiDocs
+./gradlew hooks checkstyleMain checkstyleTest asciidoctor
+
+# Store the last exit code in a variable
+RESULT=$?
+
+if [ $RESULT -eq 0 ]
+then
+    echo -e "\n================ Pre-commit checks: PASSED ================\n"
+else
+    echo -e "\n================ Pre-commit checks: FAILED ================"
```

Comment:
missing one \n at the end

Reply:
Oh dank thanks! Oddly, the output is a bit botched on Windows, but it's Windows, so...